common/protocol/Makefile.mk

```diff
@@ -21,8 +21,8 @@ common_protocol_libolaproto_la_CXXFLAGS = $(COMMON_CXXFLAGS_ONLY_WARNINGS)
 
 common_libolacommon_la_LIBADD += common/protocol/libolaproto.la
 
-common/protocol/Ola.pb.cc common/protocol/Ola.pb.h: common/protocol/Ola.proto
+common/protocol/Ola.pb.cc common/protocol/Ola.pb.h: common/protocol/Makefile.mk common/protocol/Ola.proto
```

Comment:
I seem to remember removing this because it was causing a lot of things to get rebuilt on each pass through autoconf. That doesn't make sense though as the Makefile.mk should be static.

Can you just confirm?